go.mod

```diff
@@ -5,8 +5,7 @@ go 1.18
 require (
 	github.com/celestiaorg/nmt v0.10.0
 	github.com/celestiaorg/quantum-gravity-bridge v1.2.0
-	github.com/cosmos/cosmos-sdk v0.46.0
```

Comment:
I don't like removing this line. Because,  it means adding the line below:
```
	github.com/cosmos/cosmos-sdk v0.0.0-00010101000000-000000000000
```
Which we shouldn't be depending on.

Reply:
yeah definitely, not ok dependabot

Reply:
or we can just ignore this until we have the integration tests back

Reply:
I guess it's better to ignore it. I can update geth version  manually correctly in a subsequent PR or wait as you said until we re-introduce everything

Reply:
cool, closing this then